examples/validation/validate_transform.py

```diff
@@ -0,0 +1,62 @@
+"""
+Validate transforms
+===================
+
+Apply a variety of transform combinations.
+"""
+
+# sphinx_gallery_pygfx_docs = 'screenshot'
+# sphinx_gallery_pygfx_test = 'compare'
+
+from wgpu.gui.auto import WgpuCanvas, run
+import pygfx as gfx
+
+canvas = WgpuCanvas()
+renderer = gfx.renderers.WgpuRenderer(canvas)
+
+scene = gfx.Scene()
+
+geometry = gfx.box_geometry(1, 1, 1)
+material_a = gfx.MeshBasicMaterial(
+    color=(0, 0.5, 0.5), wireframe=True, wireframe_thickness=3
+)
+material_b = gfx.MeshBasicMaterial(color=(0, 1, 1))
+
+
+def create__boxes(ttype1, value1, ttype2, value2, ttype3, value3, *, state_basis):
+    box_a = gfx.Mesh(geometry, material_a)
+    box_b = gfx.Group()
+    box_c = gfx.Mesh(geometry, material_b)
+
+    for box in (box_a, box_b, box_c):
+        box.local.state_basis = state_basis
+        box.world.state_basis = state_basis
```

Comment:
This has not effect now, but it should cover the alternative code paths of  #920

Reply:
RecursiveTransform (box.world) has no state_basis property though

Reply:
Good point. Fixed.